src/components/management/terminals/thermometers-table.tsx

```diff
@@ -15,129 +15,185 @@ import GenericDataGrid from "components/generic/generic-data-grid";
 import ToolbarRow from "components/generic/toolbar-row";
 import { TerminalTemperature, TerminalThermometer } from "generated/client";
 import { getListTerminalTemperaturesQueryOptions } from "hooks/use-queries";
-import { useMemo, useState } from "react";
+import { forwardRef, useImperativeHandle, useMemo, useState } from "react";
 import { useTranslation } from "react-i18next";
 import TimeUtils from "src/utils/time-utils";
 
 type Props = {
   name: string;
   thermometers: TerminalThermometer[];
+  onDeleteDevice: (deviceIdentifier: string) => void;
+  setChangedTerminalThermometerNames: React.Dispatch<
+    React.SetStateAction<{ newName: string; thermometerId: string }[]>
+  >;
```

Comment:
This should be (I think) okay for the type and is easier to read. Try before applying.
```suggestion
  setChangedTerminalThermometerNames: (thermometerNames: { newName: string; thermometerId: string }[]) => void,
```

Reply:
This will cause type error later in the code when it's actually used. Should I keep it as it is or resolve the other typing issue now?